utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -1149,16 +1185,16 @@ inline void compare_reduction_output(Rpp64u* output, string funcName, RpptDescPt
         }
     }
 
-    std::cerr << std::endl << "Results for " << func << " :" << std::endl;
+    std::cout << std::endl << "Results for " << func << " :" << std::endl;
     std::string status = func + ": ";
     if(fileMatch == srcDescPtr->n)
     {
-        std::cerr << "PASSED!" << std::endl;
+        std::cout << "PASSED!" << std::endl;
         status += "PASSED";
     }
     else
     {
-        std::cerr << "FAILED! " << fileMatch << "/" << srcDescPtr->n << " outputs are matching with reference outputs" << std::endl;
+        std::cout << "FAILED! " << fileMatch << "/" << srcDescPtr->n << " outputs are matching with reference outputs" << std::endl;
```

Comment:
is there any reason for changing cout to cerr?

Reply:
If we use cerr the summary is printed before the cpp code printing time log to the console